### DISCUSSION TITLE
How is Battery Replaced supposed to work?

### DISCUSSION THREAD

**Discussion by stevegroom** [andrew-codechimp/HA-Battery-Notes#1180] [Q&A] [answered]
Battery Notes is installed and seems to be working correctly. Today, I replaced batteries in a Netatmo sensor. The Battery Replaced date did not update, nor could I find any way to update it manually  via configuration. How is it meant to work?

All my devices have battery replaced = unknown.

**Comment by andrew-codechimp** [comment, marked as answer]
You can either go into the standard device page and press the Battery Replaced button in Diagnostics to manually update this when you change a battery or automate it, there is an event that is fired when the battery increases that you can write a little automation for.

See the [community](https://andrew-codechimp.github.io/HA-Battery-Notes/community/#battery-replaced) section for an example automation, there's also a blueprint further down if you don't want to customise anything.

I give these options as people want to handle this differently.

Devices default to unknown until you have replaced a battery once, I can't tell when they were replaced prior to Battery Notes being setup.
